backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/comments/CommentEntryRepository.java

```diff
@@ -57,6 +67,13 @@
 	private final IQueryBL queryBL = Services.get(IQueryBL.class);
 	private final IADTableDAO tableDAO = Services.get(IADTableDAO.class);
 
+	private final CCache<TableRecordReference, Boolean> referenceHasCommentsCache = CCache.<TableRecordReference, Boolean>builder()
+			.cacheName("referenceHasCommentsCache")
+			.cacheMapType(CCache.CacheMapType.LRU)
+			.initialCapacity(10_000)
```

Comment:
that's a bit too much. IMHO 1000 would be sufficient.

Reply:
done